data/spark

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
This file is missing the .json ending. That’s why it isn’t showing up in the preview deploy.

Reply:
My bad! I'm new to GitHub and missed that part of the file name. I think I've corrected that now, but let me know if it's still not showing up. Thanks!

Reply:
I also grabbed the app icon from the App Store to upload it here, but I couldn't find where to do that. When I go to the directory you mentioned on the How to Contribute page and click the Upload File button, I get a message telling me that I don't have permission to upload files there. Have I missed something?